package.json

```diff
@@ -36,15 +39,19 @@
     "@babel/preset-env": "^7.12.11",
     "@babel/preset-react": "^7.12.10",
     "babel-loader": "^8.2.2",
+    "babel-plugin-add-module-exports": "^1.0.4",
+    "cross-env": "^7.0.3",
     "html-webpack-plugin": "^3.2.0",
     "jest": "^26.6.3",
     "react": "^16.14.0",
     "react-dom": "^16.14.0",
+    "rimraf": "^3.0.2",
     "webpack": "^4.44.2",
     "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0"
   },
   "dependencies": {
-    "@babel/runtime": "^7.17.9"
+    "@babel/runtime": "^7.17.9",
+    "@types/react": "^18.0.0"
```

Comment:
Shouldn't these be `devDependencies`?
* @types/react
* @babel/runtime

Reply:
the user needs to have access to the types, so in this case it's a dependency
I've also doubted, so I've investigated a bit, and it's the right way to do
@babel/runtime is also the dependency. It's in the doc: https://babeljs.io/docs/en/babel-runtime
and as far as I remember, we had the issue with it when it was a devDependency